components/RevisionTimeline/RevisionTimeline.tsx

```diff
@@ -0,0 +1,42 @@
+import { format } from 'date-fns';
+import { Submission } from 'data/flexibleForms/forms.types';
+
+interface Props {
+  submission: Submission;
+}
+
+const RevisionTimeline = ({ submission }: Props): React.ReactElement | null => {
+  // reverse the array so it's in reverse-chronological order and take the three most recent events
+  const revisions = submission.editHistory.concat([]).reverse().slice(0, 3);
```

Comment:
Are we using .concat only to create a new array object?

I think `Array.from(submission.editHistory).reverse().slice(0, 3);` is cleaner and the code shows the intention